src/Controller/AdminController.php

```diff
@@ -53,7 +53,7 @@
 use AppBundle\Form\PricingRuleSetType;
 use AppBundle\Form\RestaurantAdminType;
 use AppBundle\Form\SettingsType;
-use AppBundle\Form\StripeLivemodeType;
```

Comment:
in the issue #3042 it's say
> use Stripe **OR** PayGreen

if replacing Stripe by PayGreen, Stripe no more available.